packages/components/progress/src/less/progress.less

```diff
@@ -168,6 +192,17 @@
   }
 }
 
+.segment-colors(@i: length(@oui-progress-segment-colors)) when (@i > 0) {
+  .oui-progress__bar_info:nth-of-type(@{i})::before {
+    @name: extract(@oui-progress-segment-colors, @i);
```

Comment:
What will happen if we have more than 13 progress-bar ? I mean, should it have the first color if it goes beyond 13 ? Using a `%` operator could do the job, what do you think ?

Reply:
Agree. It seems that there should be no situations whatsoever to have more than 13 segments in a chart, but % will duplicate a specific color, leading to a balagan for legends obviously...